pkg/gcp/gcs/gcs.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gcs
+
+import (
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"k8s.io/release/pkg/command"
+	"k8s.io/release/pkg/gcp"
+)
+
+var (
+	gcsPrefix      = "gs://"
+	concurrentFlag = "-m"
+	recursiveFlag  = "-r"
+)
+
+func CopyToGCS(src, gcsLocation string, recursive, concurrent bool) error {
+	gcsLocation = strings.TrimPrefix(gcsLocation, "gs://")
+	gcsLocation = gcsPrefix + gcsLocation
+
+	logrus.Infof("Copying %s to GCS (%s)", src, gcsLocation)
+	return copy(src, gcsLocation, recursive, concurrent)
+}
+
+func CopyToLocal(gcsLocation, dst string, recursive, concurrent bool) error {
+	gcsLocation = strings.TrimPrefix(gcsLocation, "gs://")
+	gcsLocation = gcsPrefix + gcsLocation
+
+	logrus.Infof("Copying GCS (%s) to %s", gcsLocation, dst)
+	return copy(gcsLocation, dst, recursive, concurrent)
+}
+
+func copy(src, dst string, recursive, concurrent bool) error {
+	args := []string{}
+
+	if concurrent {
+		args = append(args, concurrentFlag)
+	}
+
+	args = append(args, "cp")
+	if recursive {
+		args = append(args, recursiveFlag)
+	}
+	args = append(args, src, dst)
+
+	cpErr := command.New(
+		gcp.GSUtilExecutable,
+		args...,
+	).RunSuccess()
```

Comment:
Do we wanna use 
```suggestion
	cpErr := command.Execute(
		gcp.GSUtilExecutable,
		args...,
	)
```
here?